src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/steps/DeletedAccountsGetApiSteps.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.laubackend.idam.serenityfunctionaltests.steps;
+
+import net.thucydides.core.annotations.Step;
+import org.json.JSONException;
+import uk.gov.hmcts.reform.laubackend.idam.serenityfunctionaltests.model.DeletedAccountsRequest;
+import uk.gov.hmcts.reform.laubackend.idam.serenityfunctionaltests.model.DeletedAccountSearch;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DeletedAccountsGetApiSteps extends BaseSteps {
+
+    @Step("Given the GET service body is generated")
+    public DeletedAccountsRequest generateDeletedAccountsRequest() {
+        DeletedAccountSearch deletedAccountSearch = DeletedAccountSearch.builder()
+            .userId("user id")
+            .emailAddress("email@example.net")
+            .firstName("John")
+            .lastName("Smith")
+            .startTimestamp("2023-05-23T09:34:23.432Z")
+            .endTimestamp("2023-06-24T09:34:23.432Z")
+            .build();
+
+        DeletedAccountsRequest request = new DeletedAccountsRequest();
+        request.setDeletionSearchLogs(Arrays.asList(deletedAccountSearch));
+        return request;
+    }
+
+    @Step("Given invalid GET body is generated")
+    public DeletedAccountsRequest generateInvalidDeletedAccountsSearchRequest() {
+        DeletedAccountSearch deletedAccountSearch = DeletedAccountSearch.builder()
+            .userId("user id")
+            .emailAddress("")
+            .firstName("John")
+            .lastName("Smith")
+            .endTimestamp("2023-05-23T09:34:23.432Z")
+            .build();
+
+        DeletedAccountsRequest request = new DeletedAccountsRequest();
+        request.setDeletionSearchLogs(Arrays.asList(deletedAccountSearch));
+        return request;
+    }
+
+    @Step("And valid Authorization token is generated")
+    public String validAuthorizationTokenIsGenerated() throws JSONException {
```

Comment:
I believe, you don't need this as you have exactly the same method in base class that you extend.

Reply:
Done